db/queries/queries.py

```diff
@@ -0,0 +1,48 @@
+import psycopg
+import os
+
+NACHET_DB_URL = os.getenv("NACHET_DB_URL")
+
+NACHET_SCHEMA = os.getenv("NACHET_SCHEMA")
+
+def createConnection():
```

Comment:
A trick in Python is you can use is using the dunder function `__exit__` and `__enter__`, by configuring those, you enable the use of the keyword `with`, which will help your package to be used in a more readable way. If you want you can turn this module into a class and use `with` in all other classes. It will assure you that you always enter and exit gracefully the connection to the database.

Reply:
noted. But this module wont be used. It was more to structure my query functions as I was creating them.